frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -52,7 +52,8 @@ end
 
 function InputContainer:paintTo(bb, x, y)
     if not self.dimen then
-        self.dimen = self[1]:getSize()
+        local content_size = self[1]:getSize()
```

Comment:
Yes, but some widgets return a simple table with width and height in getSize method. It's true that we should use Geom in these methods.